src/classes/User.js

```diff
@@ -49,6 +50,28 @@ class User {
 
     return missingIngredients
   }
+
+  cookRecipe(recipe) {
+    recipe.ingredients.forEach(ingredient => {
+      fetch('http://localhost:3001/api/v1/users'), {
+        method: 'POST',
+        headers: {'Content-Type': 'application/json'},
+        body: JSON.stringify({'userID': this.id,
+        'ingredientID': ingredient.id,
+        'ingredientModification': ingredient.amount * -1})
+      }
+      .then((response) => {
+        if (response.status >= 200 && response.status <= 299) {
+          const pantryIndex = this.pantry.findIndex(pantryIngredient => pantryIngredient.id === ingredient.id)
+
+          this.pantry[pantryIndex].amount -= ingredient.amount
```

Comment:
We would need to `parse` the response in order to get access to the server response.